frontend/src/components/Map/Map.tsx

```diff
@@ -50,23 +52,31 @@ const Map: React.FC<PropsType> = props => {
           attribution='&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
           url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
         />
-        {props.points !== undefined &&
-          props.points.map(
-            point =>
-              point.location !== null && (
-                <Marker
-                  key={point.id}
-                  position={[point.location.y, point.location.x]}
-                  icon={isSelectedMarker(point.id) ? ActiveTrekMarker : TrekMarker}
-                >
-                  <Popup
-                    id={point.id}
-                    handleOpen={() => setSelectedMarkerId(point.id)}
-                    handleClose={resetSelectedMarker}
-                  />
-                </Marker>
-              ),
-          )}
+        {/* https://github.com/Leaflet/Leaflet.markercluster#all-options */}
+        <MarkerClusterGroup
+          iconCreateFunction={ClusterMarker}
+          maxClusterRadius={(zoom: number) => (zoom <= 13 ? 40 : 20)}
```

Comment:
On peut mettre ça dans des constantes pour que ce soit plus explicit

Reply:
Carrément, c'était un peu trop quick and dirty là 😅